src/components/LeftNav.js

```diff
@@ -57,7 +59,7 @@ const TopicList = ({ tags, slug }, { dispatch }) => {
     {!isEmpty(followed) && followed.map(tag =>
       <TagLink name={tag.name} key={tag.name} highlight={tag.new_post_count} />)}
     {slug && <li>
-      <a onClick={() => dispatch(showAllTags(slug))} className='browse-all'>
+      <a onClick={() => dispatch(showAllTags(community))} className='browse-all'>
```

Comment:
I think we should connect this component, or better, pass a showMoreTopicsHandler function as a prop from a connected component higher up. To get things into a testable state we need to start connecting and mapDispatchToProp'ing in all cases where we have been using dispatch within the component. At that point we have a unit testable component with the right API in place and later we can go back and address any unnecessary state coupling in our architecture by looking at these small components being connected to the store on a case by case basis. So in this case, this is what I'd do:

`(so in the mapDispatchToProps you'd showMoreTopicsHandler: () => showAllTags(community))`